rails.gemspec

```diff
@@ -27,6 +27,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'activejob',     version
   s.add_dependency 'railties',      version
 
-  s.add_dependency 'bundler',         '>= 1.3.0', '< 2.0'
+  s.add_dependency 'bundler',         '>= 1.9.0', '< 2.0'
```

Comment:
Right. I still think that we don't need to bump the dependency here since it is not a hard dependency. What do you think about leaving this version requirement with 1.3.0 and if users get that message they can just upgrade?